presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlDistributedQueries.java

```diff
@@ -44,12 +47,6 @@ protected QueryRunner createQueryRunner()
                 TpchTable.getTables());
     }
 
-    @AfterClass(alwaysRun = true)
-    public final void destroy()
-    {
-        postgreSqlServer.close();
```

Comment:
> That led to situation that there was a time where query runner was up
and running but without a dependant service.

Was it a problem?

Reply:
It is not a problem Postgres testing, it does not manifest itself anyhow. It is a generic problem with resources in ATQF, and Postgres tests are just example.